recipes/arg_router/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, rmdir
+from conan.tools.cmake import CMakeDeps, CMakeToolchain, CMake
+
+
+class arg_routerRecipe(ConanFile):
+    name = "arg_router"
+    license = "BSL-1.0"
+    author = "Camden Mannett"
+    url = "https://github.com/cmannett85/arg_router"
+    description = "C++ command line argument parsing and routing."
+    topics = ("cpp", "command-line", "argument-parser", "header-only")
+
+    requires = "boost/1.74.0", "span-lite/0.10.3"
+
+    settings = "build_type", "compiler"
```

Comment:
This is a touchy one, since some of the requirements may use the settings it's generally just easier to leave them in.

```suggestion
    settings = "os", "arch", "compiler", "build_type"
```